config/locales/email_subscribers.es.yml

```diff
@@ -0,0 +1,27 @@
+es:
+  email_subscribers:
+    form:
+      subscribe_description: "Suscríbete para recibir un resumen semanal de las publicaciones en tu bandeja de entrada"
+      email_placeholder: "Introduce tu email..."
+      submit_button: "Suscribirse"
+    create:
+      success_message: "Gracias por suscribirte. A menos que ya estés suscrito, un correo de confirmación está en camino a %{email}"
+    confirmations:
+      success_message: "Tu suscripción a %{blog_name} está confirmada. Recibirás un resumen semanal por correo cuando se publiquen nuevas publicaciones."
```

Comment:
This one is correct but reads a bit repetitive because it says something like  "when new _publications_ are published" (translating "posts" to _publicaciones_ is correct, but the final result is very robot-like). I think a more natural sentence would be "cuando se publiquen nuevos artículos" ("artículo" would be article in reality, but it's also used for post, and I've seen the French version uses articles, which totally works in Spanish as well).

Reply:
Updated - thanks so much! 😊